### PR TITLE
VACMS-12707: Set up content entities for translation

### PR DESCRIPTION
**Pull request by timcosgrove**
## Description

Relates to #12707

### Code review
This PR will be difficult if not impossible to review from a code perspective. There are over 2400 files, most of which are configuration. Fortunately, most of these changes are triggered the first time Content Translation is enabled for any entity in a Drupal instance. Subsequent translation setup should produce small PRs.

I would appreciate if folks would review the documentation file, to see if it is understandable. Thankfully it's right at the top of the list of files.

### Settings checkup
This spreadsheet contains all the entities that have been configured for translation in this PR, along with indications of what fields are meant to be translatable or not. https://docs.google.com/spreadsheets/d/1OTnXEPMWC6Dne7FFqXAlkuwh2IPo1MYVrSYLKzP41VQ/edit#gid=171350482

The configuration of these settings can be seen in the Content Language and Translation admin page: https://cms-9gbn9g9jbv4z7x8psploheum4vcyx685.ci.cms.va.gov/admin/config/regional/content-language

It is too much to verify the settings of every one of these entities - there are 44 of them - but it would be helpful to pick 5-7 and verify that the settings match the spreadsheet. In particular, the core content types listed below and their paragraphs are most important:
* Benefits Detail Page
* Benefits Hub Page
* FAQ
* Resources and Support Detail Page

### Manual translation
1. As user with "content admin" role, visit the Content Admin overview https://cms-9gbn9g9jbv4z7x8psploheum4vcyx685.ci.cms.va.gov/admin/content
2. Filter the content to one of the core content types. Pick any piece of content to view.
  - [x] You should see a translate tab with the main tabs
3. Open the Translate tab in a new window, so you'll be able to compare the translation with the source.
  - [x] You should see the option to add a Spanish version and a Tagalog version. 
4. Click add Spanish.
  - [x] You should see a form similar to a normal node form, with English content filled in. 
5. Compare the two forms. In most cases, the forms should differ; the Spanish translation form should have fewer form fields. Form fields that are not translatable will not show on the Spanish form. Paragraphs are an exception: You will see Paragraph form fields in the translation form.
  - [x] Compare the forms and ensure there are no fields in the Spanish form that should not be there.
6. Fill out the Spanish form. It's not important to actually translate it into Spanish; it's just important you can identify the changes. When you're done, save.
  - [x] Compare the two pages. The Spanish should match what you entered, and should present as a translation of the English.

Try a few other pieces of content, from different content types, with the above steps.

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `CMS Team`
- [ ] `Public websites`
- [ ] `Facilities`
- [ ] `User support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [x] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: READMES/multilingual.md
Comment (ndouglas) [inline READMES/multilingual.md:1]: I see a couple of grammatical issues but the writing is clear and I don't see any reason to hold this up, even for a moment.

File: tests/cypress/integration/translation.feature
Comment (ndouglas) [inline tests/cypress/integration/translation.feature:1]: Thanks for the test.

**Comment by timcosgrove** [comment]
> Screenshot 2023-03-15 at 9 36 27 AM
> 
> Received this trying to add Spanish here:
> 
> https://cms-9gbn9g9jbv4z7x8psploheum4vcyx685.ci.cms.va.gov/es/node/34313/translations/add/en/es#undefined
> 
> Is that expected? Benefits Landing Page worked fine.

I looked into this and found issues with Markup fields being marked as non-translatable. I suspect this is a bug with the module, but I did not find any open issues addressing it. I made the Markup fields translatable, which solves the given problem, and I will investigate the potential issue with the Markup module separately.

**Comment by timcosgrove** [comment]
> Screenshot 2023-03-17 at 8 16 17 AM
> 
> I think we need to update the Recent changes and preview blocks so that they don't show on the add-translation pages.

Fixed. Let's do other features as follow-on tickets.

**Review by ndouglas — APPROVED** [review]
Yup, everything looks right.  You've warmed this old man's cold, dead heart.

**Pull request merged**